pontoon/base/formats/lang.py

```diff
@@ -147,6 +149,10 @@ def visit_entity(self, node, (string, translation)):
             tags = [m.group(1) for m in tag_matches]
             translation = translation[:tag_matches[0].start()].strip()
 
+        if translation == '':
+            raise Exception('Blank translation for key {key} is not allowed in '
+                            'langfiles.'.format(key=string))
+
```

Comment:
True, but this gets wrapped by a `ParseException` that includes the filename, so we'll know which locale failed and whether it's a source file or not. Also, technically, the strings in the source files are still translations, they just happen to be to the same language.

But honestly it's just that I don't want to have to add some logic for a conditional message and it won't help us debug any better, so I want to be lazy. :P